python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -222,6 +224,18 @@ def k8s_api_retry_creation_mutation(
                     k8s_api_exception=e,
                     original_exc_info=sys.exc_info(),
                 ) from e
+        except urllib3.exceptions.HTTPError as e:
+            # Temporary for recovery detection
+            print(f"k8s_api_retry_creation_mutation: {e.__module__}.{e.__class__.__name__}: {e!s}")  # noqa: T201
```

Comment:
could you update this with `logger.getLogger(__name__)` at the top of the module and use the logger instead? otherwise LGTM